src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingEnchanter.java

```diff
@@ -0,0 +1,360 @@
+package com.minecolonies.coremod.colony.buildings.workerbuildings;
+
+import com.minecolonies.api.colony.ICitizenData;
+import com.minecolonies.api.colony.IColony;
+import com.minecolonies.api.colony.IColonyView;
+import com.minecolonies.api.colony.buildings.ModBuildings;
+import com.minecolonies.api.colony.buildings.registry.BuildingEntry;
+import com.minecolonies.api.colony.jobs.IJob;
+import com.minecolonies.api.items.ModItems;
+import com.minecolonies.api.util.BlockPosUtil;
+import com.minecolonies.api.util.NBTUtils;
+import com.minecolonies.blockout.views.Window;
+import com.minecolonies.coremod.MineColonies;
+import com.minecolonies.coremod.client.gui.WindowHutEnchanter;
+import com.minecolonies.coremod.colony.buildings.AbstractBuildingWorker;
+import com.minecolonies.coremod.colony.jobs.JobEnchanter;
+import com.minecolonies.coremod.network.messages.EnchanterWorkerSetMessage;
+import io.netty.buffer.ByteBuf;
+import it.unimi.dsi.fastutil.ints.AbstractInt2BooleanMap;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.Tuple;
+import net.minecraft.util.math.BlockPos;
+import net.minecraftforge.common.util.Constants;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static com.minecolonies.api.util.constant.Constants.STACKSIZE;
+import static com.minecolonies.api.util.constant.NbtTagConstants.TAG_POS;
+
+/**
+ * The enchanter building.
+ */
+public class BuildingEnchanter extends AbstractBuildingWorker
+{
+    /**
+     * Nbt tag for the list of buildings to gather.
+     */
+    private static final String TAG_GATHER_LIST = "buildingstogather";
```

Comment:
Constants file, also capitalisation doesn’t match that of other NBT tags

Reply:
urgs